### PR TITLE
[PE-1065] Add data source connections to infra

### PR DESCRIPTION
**Pull request by kevinkirkup**
Add configuration for deploying the Pureport Environment as part of the test infrastructure.
That should save us some money.

### CODE REVIEW

File: test-infr/modules/azure-express-route/outputs.tf [hunk 1]
```diff
@@ -1,5 +1,5 @@
-output "express_route_service_key" {
-  description = "Expressroute Service Key"
-  value       = [azurerm_express_route_circuit.main.service_key]
+output "datasource_express_route" {
+  description = "Expressroute Service Key for DataSource Testing"
```
Comment (Try75) [inline test-infr/modules/azure-express-route/outputs.tf]: Is this still a service key?

File: test-infr/modules/pureport-infra/main.tf [hunk 2]
```diff
@@ -0,0 +1,43 @@
+data "pureport_accounts" "terraform_acceptanc_tests" {
```
Comment (Try75) [inline test-infr/modules/pureport-infra/main.tf]: Not sure it matters, but missing the "e" on acceptance.

File: test-infr/modules/pureport-infra/production/variables.tf [hunk 3]
```diff
@@ -0,0 +1,15 @@
+variable "connections_network_href" {
+  description = "The HREF for the network we should be deploy the connections"
+}
+
+variable "datasource_network_href" {
+  description = "The HREF for the network we should be deploy the connections"
+}
+
+variable "datasource_express_route" {
+  description = "The Express Route Service Key to use for DataSource Testing"
```
Comment (Try75) [inline test-infr/modules/pureport-infra/production/variables.tf]: This is still service key, too?

File: test-infr/modules/azure-express-route/outputs.tf (hunk 1 above)
Comment (kevinkirkup) [reply test-infr/modules/azure-express-route/outputs.tf]: Fixed.

File: test-infr/modules/pureport-infra/main.tf (hunk 2 above)
Comment (kevinkirkup) [reply test-infr/modules/pureport-infra/main.tf]: Updated.

File: test-infr/modules/pureport-infra/production/variables.tf (hunk 3 above)
Comment (kevinkirkup) [reply test-infr/modules/pureport-infra/production/variables.tf]: Changed to be consistent across variables.

**Pull request merged**